Test/Pester5.Tests.ps1

```diff
@@ -437,7 +260,24 @@ Describe 'PassingTests' {
 '@ | Set-Content -Path $Configuration.TestResult.OutputPath
             return ([pscustomobject]@{ 'TestResult' = [pscustomobject]@{ 'Time' = [TimeSpan]::Zero } })
         }
-        WhenPesterTaskIsInvoked
+        WhenPesterTaskIsInvoked -WithArgument @{

```

Comment:
Since you're not actually running Pester, you don't need to pass anything to `WithArgument` but an empty hashtable. Assert that the `Invoke-Command` mock was called with expected parameters (ArgumentList).